app.js

```diff
@@ -28,6 +28,10 @@ app.locals.title = `${capitalized(projectName)}- Generated with Ironlauncher`;
 const index = require('./routes/index');
 app.use('/', index);
 
+const authRoutes = require('./routes/auth.routes');
+app.use('/', authRoutes);
+
+
 // ❗ To handle errors. Routes that don't exist or errors that you handle in specific routes
 require('./error-handling')(app);
 
```

Comment:
remember that you have to require the session config on this file for it to work.

`require('./config/session.config')(app);`